release.config.js

```diff
@@ -14,6 +14,7 @@ const git = [
       'architect-yml.md',
       'src/dependency-manager/schema/architect.schema.json',
     ],
+    'message': `chore(release): \${nextRelease.version} ${branch === 'main' ? '' : '[skip ci]'}\n\n\${nextRelease.notes}`,
```

Comment:
I don't think this will fix - since we want the gha to run before we merge into main

Reply:
Oh yeah I see what you mean. That's weird then. We either need to omit those commits, which feels wrong, or update all of the `chore(release)` messages

Reply:
Doesn't look like we can include a manual step here if `[skip ci]` is also present

https://github.com/architect-team/architect-cli/pull/830

Reply:
Have any other ideas?